app/src/main/java/com/omgodse/notally/activities/NotallyActivity.kt

```diff
@@ -82,17 +82,13 @@ abstract class NotallyActivity(private val type: Type) : AppCompatActivity() {
         setContentView(binding.root)
 
         lifecycleScope.launch {
-            if (model.isFirstInstance) {
```

Comment:
I'm not really sure why this was introduced in the first place, perhaps some minor performance improvement.
But it prevented the the Views from being properly re-created after changing the language of the app.